modulemd/tests/ModulemdTests/modulestream.py

```diff
@@ -643,6 +647,14 @@ def test_upgrade(self):
 """,
         )
 
+    def test_upgrade_v2_to_v3(self):
+        # TODO: implement test
+        pass
+
+    def test_upgrade_v1_to_v3(self):
+        # TODO: implement test
+        pass
+
```

Comment:
If the upgrade isn't implemented yet, you should probably leave the test out of the PR.

Reply:
These were placeholders to make sure I didn't forget about them. I'll include them with my upgrade work instead. I also dropped the corresponding C versions of the placeholders from this PR.